chroma_core/models/jobs.py

```diff
@@ -350,6 +352,9 @@ def __hash__(self):
 
     wait_for_json = models.TextField()
     locks_json = models.TextField()
+    class_name = models.TextField(default="")
+    description_out = models.TextField(default="")
+    cancellable_out = models.BooleanField(default=True)
```

Comment:
Why can't we just convert cancellable to a model field instead of adding a copied field?  Does that not work somehow?

Reply:
There are original method `description` and field `cancellable` in the `Job` (and all the descendants). I've made sure not to interfere with them.